client/components/account-status/account-tools/index.tsx

```diff
@@ -24,13 +24,15 @@ const handleReset = () => {
 	trackAccountReset();
 
 	window.location.href = addQueryArgs( wcpaySettings.connectUrl, {
-		'wcpay-reset-account': true,
+		'wcpay-reset-account': 'true',
+		source: 'wcpay-reset-account', // Overwrite any existing source because we are starting over.
 	} );
 };
 
 export const AccountTools: React.FC< Props > = ( props: Props ) => {
+	const source = 'wcpay-finish-setup-tool';
 	const accountLink = addQueryArgs( props.accountLink, {
-		source: 'account-tools__finish-setup-button',
```

Comment:
I wonder if it'll affect our funnels somehow, because the source changes
I guess it will, probably data teams will need to do some tweaks, right?

Reply:
I wouldn't worry about it as I am pretty sure nobody took that source value into account. We are currently focused on the big sources (Woo core ones and our Connect page).